### ISSUE TITLE
Cannot Install Slyguy Kodi Repository

### ISSUE THREAD

**Issue by metalpatton** [matthuisman/slyguy.addons#555]
I have been a long time user of the ESPN 3 add-on from the official Kodi add-on repository. It is no longer working for me so I have been looking for a replacement. I read good things about your ESPN add-on so I thought I would try that. Although I am not able to successfully add the SlyGuy repository from following the instructions here: https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html. I am using Kodi 18.9 (Leia).

Here are the steps I'm doing:

1. Navigate to Settings -> System
2. Make sure Add-ons -> Unknown Sources is Enabled
3. Navigate to Settings -> File manager
4. Select Add source
5. Select <None> and type in https://slyguy.uk
6. Select OK
7. Enter SlyGuy as the source name
8. Select OK
9. Navigate to Settings -> Add-ons
10. Select Install from zip file
11. Select SlyGuy

At this point nothing happens. The Kodi UI doesn't change and I don't see the **repository.slyguy.zip** file shown like it is in this screenshot: https://1.bp.blogspot.com/-UKC5toSbP_Q/Xk2yLbUrBaI/AAAAAAAAFZM/eBs_6CoibE44EhNoZxpFpKEkEvjp6qomgCLcBGAsYHQ/s1600/screenshot004.png

Any ideas what might be happening or what I could be doing wrong?

Thanks in advance for any help.

**Comment by matthuisman** [comment]
If you open that url in a browser on another device, can you see the zip file?

**Comment by metalpatton** [comment]
Just using my web browser, I can view https://slyguy.uk and see the zip file. But you have a good suggestion... I should try it on another device with Kodi. I'll try that soon and will respond here.

**Comment by matthuisman** [comment]
Or download the zip and put on a USB drive or something and install it that
way.

But I'd think maybe typed into Kodi wrong? Should work

**Comment by metalpatton** [comment]
I tried it a different PC and different network today at my work but oddly I was seeing the same issue there. When I looked at the kodi.log, it seemed to be complaining about certificate issues:

2023-09-16 00:14:20.631 T:8464   ERROR: CCurlFile::FillBuffer - Failed: Peer certificate cannot be authenticated with given CA certificates(60)
2023-09-16 00:14:20.631 T:8464   ERROR: CCurlFile::Open failed with code 0 for https://slyguy.uk/.repo/addons.xml.md5:
2023-09-16 00:14:20.631 T:8464   ERROR: XFILE::CFileCache::Open - failed to open source <https://slyguy.uk/.repo/addons.xml.md5>
2023-09-16 00:14:20.631 T:8464   ERROR: CRepository: failed read 'https://slyguy.uk/.repo/addons.xml.md5'

After some digging, it appears the trusted certificate authorities were out of date in system/certs/cacert.pem. I downloaded a new cacert.pem from https://curl.se/docs/caextract.html and that fixed the issue!

Thanks for your patience and help as I worked through this. Much appreciated!
